tabcmd/commands/user/user_data.py

```diff
@@ -1,34 +1,77 @@
+import argparse
+import logging
 from enum import IntEnum
-
+from typing import List, Callable, Optional
 from tabcmd.commands.server import Server
```

Comment:
Do you still need this import?

Reply:
yes, although I may be able to get rid of it with some more refactoring.